vizro-ai/docs/pages/user-guides/dashboard-examples.md

```diff
@@ -0,0 +1,195 @@
+# Gallery of examples
```

Comment:
I'm slightly worried about confusing the readers -- we keep our gallery of Vizro examples linked from vizro.mckinsey.com but we have this gallery here in docs and it's not linking out to anywhere to try them (the gallery is just a few screenshots). I know it's harder with PyCafe etc because we need API keys but I wonder if we need to think about consistency and have these examples on the mckinsey.vizro.com site with screenshots and links to PyCafe where people can add their own API key to try them out.

@maxschulz-COL WDYT?

Reply:
Hmm you raise a very valid point, it is indeed inconsistent. And I am not sure what I would propose here.

Regarding your suggestion, I am not sure that this would flow nicely, where on vizro.mckinsey.com would this fit nicely 🤔 ?

Alternative but slightly contentious suggestion:
- scrap the entire `Examples` section for a tab that says: `Try out!` (this would lead to vizro.mckinsey.com)
- add the chart examples under [Advanced charts](https://vizro--867.org.readthedocs.build/projects/vizro-ai/en/867/pages/user-guides/create-advanced-charts/) (basically as a kind of user guide), each chart in its own category
- combine [Generate a complex dashboard](https://vizro--867.org.readthedocs.build/projects/vizro-ai/en/867/pages/user-guides/create-complex-dashboard/) and [Retrieve code for a generated dashboard](https://vizro--867.org.readthedocs.build/projects/vizro-ai/en/867/pages/user-guides/retrieve-dashboard-code/) into a new `Run vizro_ai.dashboard` to match exactly what we have for charts, significantly reducing the repeat on the code generation page, which always annoyed me (this would make the two sections align), ideally using a very! simple dashboard prompt
- add the more complex dashboard examples under a new page that mimics the [Generate a complex dashboard](https://vizro--867.org.readthedocs.build/projects/vizro-ai/en/867/pages/user-guides/create-complex-dashboard/) one
- (bonus) remove the component table on the home page (the one that says whats possible) and add it in the new `Run vizro-ai.dashboard`, where it really belongs
- (bonus) Add a similarly awesome gif to the chart creation one for the dashboard creation on both the homepage and the README

Reply:
I like this suggestion! Of course, I'm not the one doing the work, but I think it would be more useful for a reader. I think we need to put the effort into Vizro-AI in coming weeks to get it in front of as many people as we can, so yes, let's consider this as a better way forward. Sorry @nadijagraca it is a chunk more effort and I totally recognise what you've put together here already. If you want, we could drop this in for now and then roll it back when @maxschulz-COL's suggested approach is ready. But I think we should just push forward and do it without a temporary version (ie. close this PR without merging). Please can I leave you to consider which you want to do?

Reply:
If you want a temporary solution, let me know and I'll do a light edit on the copy you've created.

Reply:
I also think its best if we do it the right (suggested) way, without the temporary version. 
I will close this PR and start working on the suggested changes today.